src/wizard/internal.tsx

```diff
@@ -0,0 +1,156 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useRef } from 'react';
+import clsx from 'clsx';
+import { getBaseProps } from '../internal/base-component';
+import { fireNonCancelableEvent } from '../internal/events';
+import { warnOnce } from '../internal/logging';
+import { useContainerBreakpoints } from '../internal/hooks/container-queries';
+import { useControllable } from '../internal/hooks/use-controllable';
+import WizardForm from './wizard-form';
+import WizardNavigation from './wizard-navigation';
+import { WizardProps } from './interfaces';
+import styles from './styles.css.js';
+
+import useBaseComponent from '../internal/hooks/use-base-component';
+import { useMergeRefs } from '../internal/hooks/use-merge-refs';
+import { useVisualRefresh } from '../internal/hooks/use-visual-mode';
+
+import { useInternalI18n } from '../internal/i18n/context';
+
+import { FunnelMetrics } from '../internal/analytics';
+import { useFunnel, useFunnelStep } from '../internal/analytics/hooks/use-funnel';
+import { getSubStepAllSelector } from '../internal/analytics/selectors';
+
+export default function Wizard({
```

Comment:
I may missed it...Is it used somewhere?

Reply:
It was left over from when I first did this but changed my mind to only have it in the index. After some discussion it has been moved back to internal.tsx and the internal implementation has been removed from index.